examples/medplum-scheduling-demo/src/components/AppointmentActions.tsx

```diff
@@ -89,6 +89,7 @@ export function AppointmentActions(props: AppointmentActionsProps): JSX.Element
           code: 'VR',
           display: 'virtual',
         },
+        type: [appointment.appointmentType as CodeableConcept],
         serviceType: appointment.serviceType?.[0],
```

Comment:
@rahul1 Please double-check if that's the correct mapping.

Reply:
Upon further research, I think this is not the right mapping.

in the create encounter modal, we should allow an (optional) input of the encounter type using the default bindings. https://hl7.org/fhir/R4/valueset-encounter-type.html

Reply:
Currently, the encounter is created with a simple button click. I will add a modal so the users can input the encounter type before proceeding.